packages/ibm-products/src/components/Checklist/Checklist.mdx

```diff
@@ -22,7 +22,7 @@ item's state as "not started", "in progress", and "complete".
 
 ## About Novice to pro
 
-[Novice to pro](https://pages.github.ibm.com/security/security-design/teams-across-security/novice-to-pro/overview/)
+[Novice to pro](https://pages.github.ibm.com/security/security-design/department/end-to-end-experiences/novice-to-pro/overview/)
```

Comment:
Should this link to the Security version or the PAL version?
https://pages.github.ibm.com/cdai-design/pal/novice-to-pro/overview

FYI @cameroncalder